pallets/gear/src/benchmarking/mod.rs

```diff
@@ -586,6 +594,7 @@ benchmarks! {
                     outgoing_limit,
```

Comment:
btw, you're right that changing this incoming params hurtful, couldn't it be wrapped into some other function, which will change only required params, cause most of them unchangeable from bench to bench